src/data-table/_data-table.scss

```diff
@@ -95,6 +95,22 @@
       bottom: 8px;
       left: 24px;
     }
+    
+    .mdl-data-table--sorted-ascending, .mdl-data-table--sorted-descending {
```

Comment:
These modifiers aren't quite right. The data table itself isn't modified, but an element within  a block. However, that block doesn't yet have an identifier. 

I'm thinking `mdl-data-table__header--sorted-ascending` and `mdl-data-table__header--sorted-descending` will work here, making the the block for the headings `header` (since they are headers to the columns or rows.)

cc @sgomes and @surma for more thoughts on that naming convention before going forward with it.